platform/elasticsearch/elasticsearch.go

```diff
@@ -5,23 +5,20 @@ import (
 	"fmt"
 	"github.com/olivere/elastic/v7"
 )
+
 /*
  * GetESClient() initialize elaticsearch client instance
  * and return the client
  */
 func GetESClient() *elastic.Client {
```

Comment:
Is it necessary to make this function public?

Reply:
updated the PR